srtcore/epoll.cpp

```diff
@@ -62,13 +62,18 @@ modified by
 #include "epoll.h"
 #include "logging.h"
 #include "udt.h"
+#include "logging.h"
 
 using namespace std;
 using namespace srt::sync;
 
+#if ENABLE_HEAVY_LOGGING
+static void PrintEpollEvent(ostream& os, int events);
+#endif
+// Use "inline namespace" in C++11
 namespace srt_logging
```

Comment:
No point in this comment. For very distant future?

Reply:
Likely. Old times. I'll remove it, it's useless anyway.